odoo/addons/base/models/res_partner.py

```diff
@@ -276,7 +276,7 @@ def _avatar_get_placeholder_path(self):
     @api.depends('is_company', 'name', 'parent_id.display_name', 'type', 'company_name')
     def _compute_display_name(self):
         diff = dict(show_address=None, show_address_only=None, show_email=None, html_format=None, show_vat=None)
-        names = dict(self.with_context(**diff).name_get())
+        names = dict(self.with_context(diff).name_get())
```

Comment:
What about an empty context instead?
```suggestion
        names = dict(self.with_context({}).name_get())
```

Reply:
Mostly fear that the *absence* of flags would do the wrong thing, though a check through `_get_name` indicates they're all boolean flags so it would probably be a better solution. And since we don't know what other nonsense keys overrides would add that's basically the only assumption we can make, really.

Reply:
Maybe we should "enforce" this behavior in `name_get()` by some comment/documentation:
the implementation should do _the right thing_ for `display_name` when given an empty context

Reply:
Good idea, I'll add a warning box mentioning that.